### ISSUE TITLE
[main] Accessibility page link in description differs from location

### ISSUE THREAD

**Issue by danowar2k** [moodle-an-hochschulen/moodle-theme_boost_union#818]
**Describe the bug**
The description of **theme_boost_union | accessibilitydeclarationlinkposition** says 

> "[...] you can add a link to [...]/theme/boost_union/pages/accessibility.php from anywhere in Moodle manually."

However, the accessibility page is located at theme/boost_union/accessibility/declaration.php

It is unknown as of yet whether the link is different somewhere else.

**To Reproduce**
1. Go to admin/settings.php?section=theme_boost_union_accessibility
2. Try to go to theme/boost_union/pages/accessibility.php and fail

**Expected behavior**
The link should either point to the correct URL... or the accessibility pages should be moved to the pages directory (It is unclear to me why they are in a separate folder right now)

**Please complete the following information:**
 - Moodle Version 4.1
 - Boost Union Version v4.1-r35
 - OS: Windows
 - Browser: Chrome

**Comment by danowar2k** [comment]
Okay, judging from the mass of settings, they seem to be more configurable than the other static pages (e.g. imprint). Ah well, if that is the criteria to place them there...okay.

**Comment by danowar2k** [comment]
Looks like the lang string is the only place where the page link is wrong. It uses **theme_boost_union_get_staticpage_link** which doesn't differentiate between the regular static pages and the accessibility ones.

**Comment by abias** [comment]
Thank you @danowar2k for highlighting this glitch.
It seems that I missed to replace the theme_boost_union_get_staticpage_link() call with theme_boost_union_get_accessibility_link() for this setting. 

I will fix this shortly.
